Makefile

```diff
@@ -84,7 +84,7 @@ endif
 .PHONY: $(PLATFORMS)
 
 test:
-	ulimit -n 400; go test -short -p 1 ./...
+	ulimit -n 500; go test -short -p 1 ./...
```

Comment:
wont work on mac default terminal

Reply:
but now we have more file descriptors since we run a udp server.. (especially this is needed in tests)

Reply:
also seems like it works in my mac default terminal